chain/state/statetree.go

```diff
@@ -170,7 +170,10 @@ func (st *StateTree) LookupID(addr address.Address) (address.Address, error) {
 		return address.Undef, xerrors.Errorf("loading init actor state: %w", err)
 	}
 
-	a, err := ias.ResolveAddress(&AdtStore{st.Store}, addr)
+	a, found, err := ias.ResolveAddress(&AdtStore{st.Store}, addr)
+	if err == nil && !found {
+		err = types.ErrActorNotFound
```

Comment:
Is this how we want to do this?

Reply:
Seems fine to me (we can try to propagate the `found` boolean later, for now this is less likely to break things)